src/main/java/core/basesyntax/service/OperationStrategy.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.service;
+
+import core.basesyntax.service.operations.Expense;
+import core.basesyntax.service.operations.Income;
+import core.basesyntax.service.operations.OperationService;
+import java.util.HashMap;
+import java.util.Map;
+
+public class OperationStrategy {
+    private static Map<Operation, OperationService> events = new HashMap<>(4);
+
+    static {
+        events.put(Operation.RETURN, new Expense());
+        events.put(Operation.BALANCE, new Income());
+        events.put(Operation.PURCHASE, new Expense());
+        events.put(Operation.SUPPLY, new Income());
+    }
+
```

Comment:
(only recommendation)
 I think you should initialisation this map with operation in main class

Reply:
I think filling this Map shouldn't depend on the user. The keys are predefined constants, their values also couldn't change, that's why, I think, this Map should be initialized at class load time.